pkg/signup/signup_service.go

```diff
@@ -142,15 +146,19 @@ func (s *ServiceImpl) GetSignup(userID string) (*Signup, error) {
 	if err != nil {
 		return nil, errors2.Wrap(err, fmt.Sprintf("error when retrieving MasterUserRecord for completed UserSignup %s", userSignup.GetName()))
 	}
-	murCondition, ready := condition.FindConditionByType(mur.Status.Conditions, crtapi.ConditionReady)
+	murCondition, _ := condition.FindConditionByType(mur.Status.Conditions, crtapi.ConditionReady)
+	ready, err := strconv.ParseBool(string(murCondition.Status))
+	if err != nil {
+		return nil, errors2.Wrapf(err, "unable to parse readiness status as bool: %s", murCondition.Status)
+	}
 	signupResponse.Status = Status{
 		Ready:   ready,
 		Reason:  murCondition.Reason,
 		Message: murCondition.Message,
 	}
-	if mur.Spec.UserAccounts != nil && len(mur.Spec.UserAccounts) > 0 {
-		// TODO Set TargetCluster in UserSignup.Status. For now it's OK to get it from the first embedded UserAccount from MUR.
-		signupResponse.TargetCluster = mur.Spec.UserAccounts[0].TargetCluster
+	if mur.Status.UserAccounts != nil && len(mur.Status.UserAccounts) > 0 {
+		// TODO Set ConsoleURL in UserSignup.Status. For now it's OK to get it from the first embedded UserAccount status from MUR.
+		signupResponse.ConsoleURL = mur.Status.UserAccounts[0].Cluster.ConsoleURL
```

Comment:
I would suggest that we "forge" for the `ConsoleURL` displayed in the UI here (ie, on the backend), rather than on the front-end (https://github.com/codeready-toolchain/registration-service/pull/71/files#diff-1c39f6c21128106bd25945f0d8391375R65). That would be easier to test. (Well, that's just my old habit of putting logic in the backend)

Reply:
Well. The backend provides the ConsoleURL without any modifications.
But frontend (atm) may want to adjust that URL to point user to a specific place in the console. Like specific namespace and specific view of the console (topology view for example). We will probably have more than one link in UI - dev console, che dashboard, etc.
It's not clear atm what we would like to show the user atm. So, I would just keep the base console URL in backend for now.
We might move some logic of calculating specific URL out of the console URL to backend later when we know what exactly we need.